core/modules/entity/entity.class.inc

```diff
@@ -136,6 +170,20 @@ abstract class Entity extends stdClass implements EntityInterface {
     return $this->entityType();
   }
 
+  /**
+   * Implements EntityInterface::createAccess().
+   */
+  public static function createAccess($bundle = NULL, $account = NULL) {
+    return TRUE;
+  }
+
+  /**
+   * Implements EntityInterface::access().
+   */
+  public function access($op, $account = NULL) {
+    return TRUE;
```

Comment:
Per EntityInterface docblocks: `If the entity type does not specify any access information, NULL is returned.` So these seem like they should return NULL, correct?

Reply:
Yes they should.